src/REPLMode.jl

```diff
@@ -1264,6 +1264,7 @@ The `startup.jl` file is disabled during precompilation unless julia is started
     :name => "status",
     :short_name => "st",
     :handler => do_status!,
+    :arg_count => 0 => Inf,
```

Comment:
Should add ` :arg_parser => (x -> parse_pkg(x)),` here, which will parse the arguments as packages.